### DISCUSSION TITLE
How to do context menu on table rows?

### DISCUSSION THREAD

**Discussion by AsuraKev** [ant-design/ant-design#44054] [Q&A]
### What problem does this feature solve?

Perform actions by right clicking on the table row

### What does the proposed API look like?

N/A

**Comment by kiner-tang** [comment]
You can try this way to listen onContextMenu event to show context menu.
<img width="1341" alt="image" src="https://github.com/ant-design/ant-design/assets/10286961/2b7b2ed8-3500-4749-8978-d65cd14474f7">

**Comment by AsuraKev** [comment]
Thanks :) Could you please share an example? The context menu in Antd is a Dropdown of type context menu. The onContextMenu does not allow return a ReactNode.

**Comment by AlirezaMH** [comment]
This was the only way I found, but unfortunately, it's not a clean way:
[link](https://codesandbox.io/p/sandbox/rm23kroqyo?file=%2Findex.js)
